### DISCUSSION TITLE
Add github & slack integration

### DISCUSSION THREAD

**Discussion by vladciobancai** [kimai/kimai#4585] [Questions & Answers]
Hello!

Is it possible to add "github & slack integration"(https://github.com/integrations/slack) app to kimai/kimai repo?

Link to setup https://github.com/apps/slack/installations/new

Thank you.

**Comment by kevinpapst** [comment]
Why? Which benefit des the project have?

**Comment by vladciobancai** [comment]
with github app on slack you can subscribe to this project and you can get notification of releases, issues, etc

**Comment by vladciobancai** [comment]
no need it anymore, I used feed app from slack to receive notification when a release is made
